src/app/extensions/tracking/store/tracking-config/tracking-config.effects.ts

```diff
@@ -24,7 +24,7 @@ export class TrackingConfigEffects {
     store: Store,
     cookiesService: CookiesService
   ) {
-    if (cookiesService.cookieConsentFor('tracking')) {
+    if (cookiesService.cookieConsentFor('tracking') && isPlatformBrowser(this.platformId)) {
```

Comment:
This is actually an obsolete additional check since a cookie consent will only return true `if (isPlatformBrowser(this.platformId)) {`.
The project that had SSR errors because of the missing `window` object removed the cookie condition without replacing it with a platform check. So it might be a helpful pointer even though it is not necessary?

Reply:
Changed the order of the conditions to make the importance of being in the browser more obvious and more independent from the check done by the cookies consent.